packages/cms-data-sync/package.json

```diff
@@ -19,7 +19,7 @@
     "@asap-hub/squidex": "workspace:*",
     "@babel/runtime-corejs3": "7.19.6",
     "@contentful/rich-text-types": "^15.13.2",
-    "contentful-html-rich-text-converter": "yldio/contentful-html-rich-text-converter#v1.0.14",
+    "contentful-html-rich-text-converter": "yldio/contentful-html-rich-text-converter#5fed410f9fc6f5d2771574a87ded2d0c90f7e59a",
```

Comment:
It's pointing to the last commit of this PR https://github.com/yldio/contentful-html-rich-text-converter/pull/5 but once it's merged I will change this.